aten/src/ATen/native/native_functions.yaml

```diff
@@ -1965,7 +1965,7 @@
 - func: _nnpack_available() -> bool
   use_c10_dispatcher: full
 
-- func: _nnpack_spatial_convolution(Tensor input, Tensor weight, Tensor? bias, int[2] padding) -> Tensor
+- func: _nnpack_spatial_convolution(Tensor input, Tensor weight, Tensor? bias, int[2] stride, int[2] padding) -> Tensor
```

Comment:
you are changing the signature in a backward incompatible way. 

Can you change it in the following way?
```
_nnpack_spatial_convolution(Tensor input, Tensor weight, Tensor? bias, int[2] padding,  int[2]? stride=None) -> Tensor
```
The new API is backward compatible with the existing API.

Reply:
Definitely.  Thanks for your feedback.  Just to be sure, should I unland this CR first, and update this PR to contain that fix and resubmit, or, create a new patch that contains your proposed fix only and apply that to master?

Reply:
I have already unlanded the PR and fixed the BC CI.

Please resubmit the PR and make the new aten function backward compatible. If you need any help on BC, feel free to ping me.